query-service/src/integrationTest/java/org/hypertrace/core/query/service/htqueries/HTPinotQueriesTest.java

```diff
@@ -262,21 +267,30 @@ private static boolean generateData() throws Exception {
             "service-call-view-events", 27L,
             "span-event-view", 50L,
             "log-event-view", 0L);
-    int retry = 0;
-    while (!areMessagesConsumed(endOffSetMap) && retry++ < 5) {
-      Thread.sleep(2000);
+    int retry = 0, maxRetries = 20;
+    while (!areMessagesConsumed(endOffSetMap) && retry++ < maxRetries) {
+      Thread.sleep(6000); // max 2 min wait time
```

Comment:
Somehow with the latest pinot version of servicemanager, it takes quite a time to get ready. Had to increase to `2` mins.